go.mod

```diff
@@ -324,6 +327,7 @@ require (
 )
 
 replace (
+	github.com/CosmWasm/wasmd => github.com/provenance-io/wasmd v0.29.0-pio-1
```

Comment:
let's use notional fork, it's more up to date: https://github.com/notional-labs/wasmd/tree/v0.29.0-sdk46